src/views/product-tutorials-view/components/product-view-content/components/collections-stack/types.ts

```diff
@@ -24,5 +15,5 @@ export interface CollectionsStackProps {
   subheading?: string
   /** A product slug, used for theming */
   product: ProductOption
-  featuredCollections?: CollectionStackItem[]
+  featuredCollections?: ClientCollection[]
```

Comment:
Notes from #372 probably apply here too - hoping to follow this up soon with 🎟️ [Enhancement (maybe defer): Refactor product-tutorials-view/server.ts so that we're not sending such massive amounts of data to the client (currently > 300kB for Vault!)](https://app.asana.com/0/0/1202182325935203/f)